buildbot/master/files/config/master.cfg

```diff
@@ -158,6 +177,9 @@ c['builders'] = [
         factory=factories.doc,
         category="auto",
     ),
+    # Testing the In Tree YAML build
+    DynamicServoYAMLBuilder("linux-dev-yaml", LINUX_SLAVES,
+                            envs.build_linux_headless)
```

Comment:
`envs.build_linux_headless` has been removed; use `envs.build_linux` instead. (I think it might fit on the previous line, not sure.)